Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -1666,6 +1668,9 @@ void Painter::draw_line(IntPoint const& a_p1, IntPoint const& a_p2, Color color,
                 draw_physical_pixel({ x, y }, color, thickness);
                 draw_physical_pixel({ x, min(y + thickness, max_y) }, color, thickness);
                 draw_physical_pixel({ x, min(y + thickness * 2, max_y) }, color, thickness);
+                draw_physical_pixel({ x, min(y + thickness * 3, max_y) }, alternate_color, thickness);
+                draw_physical_pixel({ x, min(y + thickness * 4, max_y) }, alternate_color, thickness);
+                draw_physical_pixel({ x, min(y + thickness * 5, max_y) }, alternate_color, thickness);
```

Comment:
This will probably regress the perf quite a bit, how about
```suggestion
                if (!alternate_color_is_transparent) {
                    draw_physical_pixel({ x, min(y + thickness * 3, max_y) }, alternate_color, thickness);
                    draw_physical_pixel({ x, min(y + thickness * 4, max_y) }, alternate_color, thickness);
                    draw_physical_pixel({ x, min(y + thickness * 5, max_y) }, alternate_color, thickness);
                }
```

where `alternate_color_is_transparent` is defined at function entry or something?